src/datasets/datasets.jl

```diff
@@ -36,6 +55,54 @@ macro load_toyfairtensor()
     end
 end
 
+"
+Macro to load COMPAS dataset.
+https://github.com/propublica/compas-analysis/blob/master/Compas%20Analysis.ipynb
+"
+macro load_compas()
+    quote
+        url = "https://raw.githubusercontent.com/propublica/compas-analysis/master/compas-scores-two-years.csv"
+        fname = "compas-scores-two-years.csv"
+        ensure_download(url, fname)
+
+        fpath = joinpath(DATA_DIR, fname)
+        data = DataFrame!(CSV.File(fpath))
+        # dropmissing!(data, names(data))
+        # coerce!(data, Textual => Multiclass)
+        # coerce!(data, :is_recid => Multiclass)
+        X = data[!, ["sex", "age", "age_cat", "race", "c_charge_degree", "priors_count", "days_b_screening_arrest", "decile_score"]]
```

Comment:
The number of columns specified here are lesser than what are present in the COMPAS Dataset.

But as this COMPAS Dataset has a lot of columns, it might be worth adding two macros for COMPAS Dataset
- @load_compas : It will have all the columns
- @load_reduced_compas : This one will have the columns which are a subset of the columns in original dataset

So, please consider renaming this macro to `load_reduced_compas` or maybe keep all the columns.